src/graphql/Displays.graphql

```diff
@@ -0,0 +1,9 @@
+type VncDisplay {
+  display: String!
+  width: Int!
+  height: Int!
+}
+
+extend type Query {
+  vncDisplays(url: String!): [VncDisplay]
```

Comment:
What is the url here?  I'm curious why none of the GraphQL is documented..

I think it would make more sense to pass the taskId.  Then the implementation would look for `displays.html` or whatever the artifact name is, and return the result.  Also, it probably makes sense to implement this along with the shell support, which should be very similar.